api/test/integration/env/configurations/rbac/webhook/white_config.txt

```diff
@@ -0,0 +1,3 @@
+| Actions                  | Resources | Allow                                   | Deny                |
+|--------------------------|-----------|-----------------------------------------|---------------------|
+| event:ingest             | *         |                                         | *                   |
```

Comment:
If you are allowing `event:ingest` in the white_config.yml, why do you say Deny here?